.gitlab-ci.yml

```diff
@@ -14,19 +14,65 @@ variables:
   BUILD_DIR: build
 
 stages:
+  - build
   - test
   - test_acceptance_build
   - test_acceptance
-  - build
   - publish
 
+compile:
+  image: golang:1.14
+  stage: build
+  before_script:
+    - mkdir -p /go/src/$(dirname $REPO_NAME)/mender-cli /go/src/_/builds
```

Comment:
`/_/` ?

Reply:
just moved around in the file - according to the green build this seems to work fine 🤷‍♂️

Reply:
I think that is some leftovers from the time where we tried to re-use GitLab cache for the builds. It can be removed)